pkg/computenodeopenstack/configmap.go

```diff
@@ -0,0 +1,29 @@
+package computenodeopenstack
+
+import (
+        util "github.com/openstack-k8s-operators/lib-common/pkg/util"
```

Comment:
wrong identation. Usually I run "gofmt -w file_name" to have the proper formating

Reply:
done